mayhem/test_CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+project(Alpha_complex_tests)
```

Comment:
Same here, this is all commented out. Or perhaps this PR is still in progress?

Reply:
No, it should be done now.

I had to comment out these targets so that the action doesn't OOM.